.ci/install_cri_containerd.sh

```diff
@@ -69,17 +71,42 @@ install_from_static_tarball() {
 	sudo tar -xvf "${tarball_name}" -C /
 }
 
+install_vanilla_from_static_tarball() {
+	echo "Trying to install containerd from static tarball"
+	local tarball_url="https://github.com/containerd/containerd/releases/download"
+	local containerd_tarball_version="v1.6.8"
```

Comment:
So I'm a bit split here:
- Either we install v1.6.8 to stay aligned with our wider kata `versions.yaml`
- Or we installing 1.7 of vanilla containerd so we can get the per runtime snapshotter setting? I think this is the version that the operator will install

I'm looking for other opinions here to decide which makes most sense